### PR TITLE
support init of buckets in SwiftStorageAdapter

### PR DESCRIPTION
**Pull request by pdowler**
create buckets  on init
store immutable config as bucket attrs in storage
fix intTest init to override some config from user config file

### CODE REVIEW

File: cadc-storage-adapter-swift/src/main/java/org/opencadc/inventory/storage/swift/SwiftStorageAdapter.java [hunk 1]
```diff
@@ -259,23 +284,72 @@ public SwiftStorageAdapter() throws InvalidConfigException, StorageEngageExcepti
...
+                    }
+                } else {
+                    log.info("exists: " + bucketName);
+                }
             }
         }
+        
+        // init succeeeded: commit
+        Map<String,Object> meta = new TreeMap<>();
+        meta.put(VERSION_ATTR, "0.5.0");
```
Comment (andamian) [inline cadc-storage-adapter-swift/src/main/java/org/opencadc/inventory/storage/swift/SwiftStorageAdapter.java:348]: What is this and how it's maintained?

File: cadc-storage-adapter-swift/src/intTest/java/org/opencadc/inventory/storage/swift/SwiftByteRangeTest.java [hunk 2]
```diff
@@ -109,4 +109,9 @@ public void cleanupBefore() throws Exception {
         }
         log.info("cleanupBefore: DONE");        
     }
+    
+    @Test
+    public void testCleanupOnly() {
+        log.info("testCleanupOnly: no-op");
```
Comment (andamian) [inline cadc-storage-adapter-swift/src/intTest/java/org/opencadc/inventory/storage/swift/SwiftByteRangeTest.java:115]: What is this for?

**Review by andamian — DISMISSED** [review]

File: cadc-storage-adapter-swift/src/main/java/org/opencadc/inventory/storage/swift/SwiftStorageAdapter.java (hunk 1 above)
Comment (pdowler) [reply cadc-storage-adapter-swift/src/main/java/org/opencadc/inventory/storage/swift/SwiftStorageAdapter.java:348]: This is the version of the library at the point where the "schema" of the storage layout is created. If we have to change the schema at some point, then we'll have to run some code to upgrade from this version to the next version and then assign the new version attr. It's the same concept I use for database schema updates (via ModelVersion table). 

I don't know if we'll ever change the "storage schema" but this at least says which version of the library first required this version.

File: cadc-storage-adapter-swift/src/intTest/java/org/opencadc/inventory/storage/swift/SwiftByteRangeTest.java (hunk 2 above)
Comment (pdowler) [reply cadc-storage-adapter-swift/src/intTest/java/org/opencadc/inventory/storage/swift/SwiftByteRangeTest.java:115]: A no-op task allows one to run just this test method so you get the ctor (init) and cleanup code and no other tests. It's handy if you are having trouble with init (connection, auth, the init itself, or the cleanupBefore code).

**Pull request merged**